modules-js/react-fleet/src/components/Header.tsx

```diff
@@ -22,3 +59,12 @@ export default function Header(
     </>
   );
 }
+
+function isSealVisible(): boolean {
```

Comment:
Make it clear that the visibility is more about breakpoints than position on the screen.